io.github.twisterarmy.Twister.json

```diff
@@ -0,0 +1,94 @@
+{
+    "app-id": "io.github.twisterarmy.Twister",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "twister",
+    "modules":
+    [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands":
+            [
+                "./bootstrap.sh",
+                "./b2 install --prefix=/app"
+            ],
+            "sources":
+            [
+                {
+                    "type": "archive",
+                    "url": "https://archives.boost.io/release/1.77.0/source/boost_1_77_0.tar.gz",
+                    "sha256": "5347464af5b14ac54bb945dc68f1dd7c56f0dad7262816b956138fc53bcc0131"
+                }
+            ]
+        },
+        {
+            "name": "libdb",
+            "buildsystem": "simple",
+            "build-commands":
+            [
+                "cd build_unix && ../dist/configure --prefix=/app --enable-cxx && make -j$(nproc) && make install"
+            ],
+            "sources":
+            [
+                {
+                    "type": "archive",
+                    "url": "https://download.oracle.com/berkeley-db/db-4.8.30.tar.gz",
+                    "sha256": "e0491a07cdb21fb9aa82773bbbedaeb7639cbd0e7f96147ab46141e0045db72a"
+                },
+                {
+                    "type": "patch",
+                    "path": "libdb.patch",
+                    "options":
+                    [
+                        "-p2"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "core",
+            "buildsystem": "simple",
+            "build-commands":
+            [
+                "./autotool.sh",
+                "./configure --prefix=/app --with-boost=/app --with-libdb=/app",
+                "make -j$(nproc) && make install",
+                "mkdir -p /app/share/icons/hicolor/scalable/apps && cp share/icons/hicolor/scalable/apps/io.github.twisterarmy.Twister.svg /app/share/icons/hicolor/scalable/apps/io.github.twisterarmy.Twister.svg",
+                "mkdir -p /app/share/applications && cp share/applications/io.github.twisterarmy.Twister.desktop /app/share/applications/io.github.twisterarmy.Twister.desktop"
+            ],
+            "sources":
+            [
+                {
+                    "type": "git",
+                    "url": "https://github.com/twisterarmy/twister-core.git",
+                    "commit": "e3fa4e9ea9a2b3f88443870fc525217d1f99c989"
+                }
+            ]
+        },
+        {
+            "name": "html",
+            "buildsystem": "simple",
+            "build-commands": [
+                "echo -e \"xdg-open http://user:pwd@localhost:28331\ntwisterd -htmldir=/app/share/html -rpcport=28331 -rpcuser=user -rpcpassword=pwd -rpcallowip=127.0.0.1\" > /app/twister",
+                "chmod +x /app/twister",
+                "ln -s /app/twister /app/bin/twister",
```

Comment:
why don't you put in bin directly?

Reply:
I don't see the relation between "core" and script you create here and install in a non standard location just to link it....